acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/pageobjects/BitbucketScmConfig.java

```diff
@@ -23,36 +23,47 @@ public BitbucketScmConfig(Job job, String path) {
     }
 
     public BitbucketScmConfig credentialsId(String credentialsId) {
+        scrollIntoView(this.credentialsId);
```

Comment:
Most of the failing acceptance tests were due to the fact that the element was obscured by something in the UI